### PR TITLE
Remove paths from markdownlint.yml

### PR DESCRIPTION
**Pull request by gewarren**
Otherwise the lint status check hangs if no .md files are touched in a PR.

### CODE REVIEW

File: .github/workflows/markdownlint.yml [hunk 1]
```diff
@@ -9,12 +9,6 @@ on:
       - ".markdownlint-cli2.jsonc"
       - ".github/workflows/markdownlint.yml"
       - ".github/workflows/markdownlint-problem-matcher.json"
-  pull_request:
```
Comment (BillWagner) [inline .github/workflows/markdownlint.yml:12]: I think we still need this line. We want it to run on PRs, right?

@IEvangelist might know more than I do.

**Review by adegeo — APPROVED** [review]

**Review by adegeo — CHANGES_REQUESTED** [review]
Do we need to just remove it from the list of "required" actions?

**Comment by adegeo** [comment]
I just spotted this in the repo for that action: https://github.com/DavidAnson/markdownlint-cli2-action/blob/main/.github/workflows/changed.yml

I think this workflow would solve the problem  and allow us to have the workflow in the list of required

**Comment by gewarren** [comment]
> I just spotted this in the repo for that action: https://github.com/DavidAnson/markdownlint-cli2-action/blob/main/.github/workflows/changed.yml
> 
> I think this workflow would solve the problem and allow us to have the workflow in the list of required

It passes like this:

image

**Review by adegeo — APPROVED** [review]
I guess if it only runs for 1 minute linting the whole repo, that's shorter than the other actions (if it's running in parallel?)

**Pull request merged**